app/views/shared/_list_of_things_sorting.html.erb

```diff
@@ -1,7 +1,15 @@
-<p class="list-of-things__sorting">
-    <%= page_entries_info @studies, entry_name: "study" %> by
-    <select>
-        <option selected>most recently updated</option>
-        <option>most recently created</option>
-    </select>
-</p>
\ No newline at end of file
+<form method="get" class="list-of-things__sorting">
```

Comment:
I think this needs one extra check to see if there are any studies, and remove the sorting completely if not - at the moment it reads all funny if there's nothing there, and we could do with a better "Nothing matched your search" message elsewhere